src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,18 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should render the new source to equal MapType', () => {
+    const newProps = {
```

Comment:
How is this test related to the ticket description?

Reply:
In the previous implementation, the reason 'relationships' data could not be changed  is because if the fields had been hardcoded in that if source was equal to CIEL, then the relationship was equal to 'SAME-AS' and if source was not equal to 'CIEL', the relationship was equal to 'NARROWER THAN'. As such, I did not have access to the 'relationship' field

One of the first steps in implementing this task was making sure that the source field in relation to the relationship field was not hardcoded but more dynamic and that is what I did with this test testing the dynamic change in the source field.